### PR TITLE
Fix broken link for testing locally on contributor-cheatsheet page

### LINKED ISSUE #6872: Link for Testing Broken on contributer cheatsheet
### Describe the issue

```
Before you propose a pull request, you will have to do some level of work locally. If you are new to git, the [Atlassian git tutorial](https://www.atlassian.com/git/tutorials) is a good starting point. As an alternative, Stanford's [Git magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/) tutorial is a good multi-language option.

References:

[Atlassian git tutorial](https://www.atlassian.com/git/tutorials)
[Git magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/)
[GitHub workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)
[Testing locally](https://github.com/kubernetes/community/blob/master/contributors/guide/README.md#testing)
[Developer guide](https://github.com/kubernetes/community/blob/master/contributors/devel/README.md)
```

here in [https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) the Testing locally doesn't poit us to the testing locally section.

### PR DESCRIPTION
**Pull request by akankshakumari393**
**Which issue(s) this PR fixes**:

Fixes #6872

### CODE REVIEW

File: contributors/guide/contributor-cheatsheet/README.md [hunk 1]
```diff
@@ -415,7 +415,7 @@ git push --force
 [GitHub workflow]: /contributors/guide/github-workflow.md#3-branch
 [squashing commits]: /contributors/guide/pull-requests.md#6-squashing-and-commit-titles
 [owners]: /contributors/guide/owners.md
-[testing locally]: /contributors/guide/README.md#testing
+[testing locally]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/testing.md
```
[the remark this answers is not in the document]
Comment (akankshakumari393) [reply contributors/guide/contributor-cheatsheet/README.md]: thanks for the reviews will take care of that from next time. 👍🏻

**Comment by jdumars** [comment]
/lgtm

thanks for fixing this!

**Review by mrbobbytables — COMMENTED** [review]
/approve

Thanks!

**Pull request merged**